spec/spec.md

```diff
@@ -307,10 +308,10 @@ Design Principle #5 ([Cryptographic Verifiability](https://trustoverip.org/perma
 
 VIDs can be divided into two subclasses as shown in figure 9:
 
-  1. **[[xref: toip, Self-certifying identifiers]] (SCIDs)** are cryptographically bound to the original key pair used to generate the VID. This means the binding between the VID and the key pair can be verified purely using cryptography—without reference to any external system. SCIDs have the advantage of being highly secure (as strong as the cryptographic algorithm used), maximally decentralized (because no external system is required), and fully portable (any external system can be used as a [[xref: toip, witness]], and the [[xref: toip, VID controller]] can change witnesses at will).
+  1. **[[xref: toip, Self-certifying identifiers]] (SCIDs)** are cryptographically bound to the original key pair used to generate the VID and subsequent key material where key rotation is supported. This means the binding between the VID and the key material can be verified purely using cryptography—without reference to any external system. SCIDs have the advantage of being highly secure (as strong as the cryptographic algorithm used), decentralized (because no external system is required), and portable.
```

Comment:
maybe not for now, but if you're going to put the advantages, why not the disadvantages?

Reply:
Again deferring